mitype/app.py

```diff
@@ -124,9 +126,24 @@ def main(self, win):
                 # Start replay if enter key is pressed
                 self.replay(win)
 
+            # Share result on Twitter
+            elif self.mode == 1 and is_ctrl_s(key):
+                # Opens twitter with pre-typed result
+                self.share_result()
+
             # Refresh for changes to show up on window
             win.refresh()
 
+    def share_result(self):
+        """Opens a twiiter on a browser"""
+
+        # Content of tweet
+        result = f"My%20typing%20speed%20is%20{self.current_speed_wpm}%20WPM!%20Know%20yours%20on%20mitype.\
```

Comment:
This is crashing in Python 2 because of f-string.
Let's keep a string say, `message` = "My typing speed is X WPM..." and then urlencode it?
This would make the code more readable and also make it easier to change later.